.yamato/ruamel/jobs/editor/editor_pinning_update.py

```diff
@@ -39,5 +39,5 @@ def get_job_definition(self, agent, target_branch, target_branch_editor_ci):
         job.add_var_custom('CI', True)
         job.add_commands(commands)
         job.add_dependencies([f'{editor_pinning_filepath()}#{editor_job_id_target_to_ci()}']) #TODO toggle
-        job.add_trigger_recurrent(target_branch, '0 * * ?') # TODO uncomment
+        job.add_trigger_recurrent(target_branch, '23 * * ?') # TODO uncomment
```

Comment:
what is this todo comment about?

Reply:
I think it was there  when the trigger was temporarily commented out. Good catch, ill remove it